hazelcast/src/main/java/com/hazelcast/cp/internal/MetadataRaftGroupManager.java

```diff
@@ -1158,6 +1165,14 @@ private boolean commitMetadataRaftGroupInit(CPMemberInfo localCPMemberCandidate,
                 raftService.destroyRaftNode(metadataGroupId);
                 return false;
             }
+
+            List<InternalCompletableFuture> futures = new ArrayList<>();
```

Comment:
Can you add a comment here, what is the reason of these random commits?

Reply:
done